pkg/registry/flowcontrol/rest/storage_flowcontrol.go

```diff
@@ -53,16 +54,25 @@ func (p RESTStorageProvider) NewRESTStorage(apiResourceConfigSource serverstorag
 	apiGroupInfo := genericapiserver.NewDefaultAPIGroupInfo(flowcontrol.GroupName, legacyscheme.Scheme, legacyscheme.ParameterCodec, legacyscheme.Codecs)
 
 	if apiResourceConfigSource.VersionEnabled(flowcontrolv1alpha1.SchemeGroupVersion) {
-		flowControlStorage, err := p.v1alpha1Storage(apiResourceConfigSource, restOptionsGetter)
+		v1alpha1FlowControlStorage, err := p.v1alpha1Storage(apiResourceConfigSource, restOptionsGetter)
 		if err != nil {
 			return genericapiserver.APIGroupInfo{}, false, err
 		}
-		apiGroupInfo.VersionedResourcesStorageMap[flowcontrolv1alpha1.SchemeGroupVersion.Version] = flowControlStorage
+		apiGroupInfo.VersionedResourcesStorageMap[flowcontrolv1alpha1.SchemeGroupVersion.Version] = v1alpha1FlowControlStorage
```

Comment:
Instead of this, I think I expected this:

```
if apiResourceConfigSource.VersionEnabled(flowcontrolv1alpha1.SchemeGroupVersion) {
  apiGroupInfo.VersionedResourcesStorageMap[flowcontrolv1alpha1.SchemeGroupVersion.Version] = v1beta1FlowControlStorage
}
```

I expected this to come after line 67 below.

It's been a while since I messed with this code, though.

Reply:
updated